### DISCUSSION TITLE
Extended example patch pointcloud material - atlas feature

### DISCUSSION THREAD

**Discussion by wirmachenbunt** [cables-gl/cables_docs#675] [Feature Requests]
there is this new atlas texture feature but i'm missing a demo, how this works in cables. i connected a 8x4 atlas texture but the settings don't make sense to me. There is only x- repeat ? And lookup doesn't seem to do anything. Fragen über Fragen :)

**Comment by pandrr** [comment]
https://dev.cables.gl/edit/Ul1pp3

**Comment by pandrr** [comment]
updated the docs
https://dev.cables.gl/op/Ops.Gl.Shader.PointMaterial_v5

**Comment by wirmachenbunt** [comment]
mmm, i'm again at the same point. how is this atlas stuff working. the example is private and the official help patch does not show how atlas works.
